### ISSUE TITLE
Relayer doesn't wait for tx confirmation

### ISSUE THREAD

**Issue by RohitAudit** [informalsystems/ibc-rs#1159]
## Crate
rellayer

## Summary of Bug

Whenever relayer signs a transaction it doesn't wait for confirmation

## Version

0.5

## Steps to Reproduce

Initially I got the error while creating channel between persistence chain and cosmos chain using following command:
```
hermes create channel cosmoshub-4 core-1 --port-a transfer --port-b transfer -o unordered
```

Following was the error I got:
```
failed during a transaction submission step to chain id cosmoshub-4 with underlying error: Failed Tx: no confirmation
```
Then after starting the relayer this error comes often:
```
Jul 07 11:02:35 relayer-mainnet-test hermes[411383]: Jul 07 11:02:35.500  INFO [core-1:transfer/channel-7 -> cosmoshub-4] prepending Destination client update @ height 1-1505507
Jul 07 11:02:40 relayer-mainnet-test hermes[411383]: Jul 07 11:02:40.420  INFO [core-1:transfer/channel-7 -> cosmoshub-4] assembled batch of 2 message(s)
Jul 07 11:02:54 relayer-mainnet-test hermes[411383]: Jul 07 11:02:54.136 ERROR failed with underlying error: Failed Tx: no confirmation
Jul 07 11:03:08 relayer-mainnet-test hermes[411383]: Jul 07 11:03:08.734  INFO [cosmoshub-4:transfer/channel-146 -> core-1] generate messages from batch with 2 events
```

But the transaction can be found on chain when the tx hash is queried. What I was able to gather from the error was that cosmos pools the transaction and commits it a bit later. Hermes doesn't wait for that long and throws this error.

## Acceptance Criteria

Is there a way to increase the timeout for transactions relayed by the relayer?

____

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate milestone (priority) applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by adizere** [comment]
Hi Rohit,

Thanks for opening an issue.

We have actually been working on this a lot recently (#1146).

Would you consider using a pre-release version of Hermes? Specifically https://github.com/informalsystems/ibc-rs/releases/tag/v0.6.0-rc2.

> Is there a way to increase the timeout for transactions relayed by the relayer?

Yes. Version 0.6.0-rc2 has two benefits:

- you can adjust `rpc_timeout = '10s'` to higher values for your `cosmoshub-4` configuration to increase the timeout
- the TxNoConfirmation error is more helpful

**Comment by RohitAudit** [comment]
Thanks adizere,

Will update hermes to the latest version.

**Comment by adizere** [comment]
Closing this as may have been fixed in the meantime. The problem seems to overlap partially with #1124 which we're still working on. 

Rohit, hope the issue has been resolved by updating to `0.6.0-rc2`, let us know otherwise.
